### DISCUSSION TITLE
Issues with "WARN  [Hugo SEO]"

### DISCUSSION THREAD

**Discussion by compscibro** [thuliteio/doks#1133] [Q&A] [answered]
Hello, I am getting the following warnings/errors after running the second command -

`npm install`
`npm run build`

Note: I have to run `npm run build` for my project. I do not have any folder named `assets` and `hugo.yaml`

```
WARN  [Hugo SEO] assets/favicon.png does not exist. Add a file that's at least 512x512 pixels (or your largest defined icon size)
WARN  [Hugo SEO] assets/mask-icon.svg does not exist.
WARN  [Hugo SEO] assets/favicon.ico does not exist.
WARN  [Hugo SEO] Missing default image for og:image - assets/cover.png does not exist. Use a high-quality image with dimensions of at least 1200 x 630 pixels, while keeping the file size under 8 MB.
WARN  [Hugo SEO] Missing default image for og:image - Add 'images: ['cover.png']' to your hugo.yaml
WARN  [Hugo SEO] Unsupported schema type: %!s(<nil>). Set Organization or Person in .Site.Params.seo.schemas.type
```

I have no clue why this is happening. I have the favicons under the `static` folder

**Comment by h-enk** [comment, marked as answer]
The favicon images need to be in the [`assets`](https://github.com/gethyas/doks/tree/main/assets) directory so that they can be processed by Hugo.

And, you need to set parameters in [`./config/_default/params.toml`](https://github.com/gethyas/seo#setup) (not `hugo.yaml`).

 I will update the warning message and add this to the documentation

**Comment by Bilal9890** [comment]
same problem
